ext/openssl/php_openssl.h

```diff
@@ -86,6 +94,13 @@ php_stream_transport_factory_func php_openssl_ssl_socket_factory;
 
 void php_openssl_store_errors();
 
+PHP_OPENSSL_API int php_openssl_cipher_iv_length(char *method, zend_long *ret);
+PHP_OPENSSL_API int php_openssl_random_pseudo_bytes(zend_string **buffer, zend_long length);
```

Comment:
I would prefer returning buffer instead. If there is an error, then it should NULL.